Tests/Admin/AdminTest.php

```diff
@@ -25,6 +26,18 @@ public function setUp()
         $this->contextManager = $this->getMock('Sonata\ClassificationBundle\Model\ContextManagerInterface');
     }
 
+    public function testAbstractAdminChildren()
+    {
+        $contextAwareAdmin = $this->getMockBuilder('Sonata\ClassificationBundle\Admin\ContextAwareAdmin')
+            ->disableOriginalConstructor()
+            ->getMock();
+        $this->assertInstanceOf(AbstractAdmin::class, $contextAwareAdmin);
```

Comment:
This will not work with old versions of php (as Travis shows). Otherwise, looks good. It will not fail if the old class is extended, but a deprecation error will be triggered, which should show up in the `SYMFONY_DEPRECATIONS_HELPER=0` build, so we're all good